op-chain-ops/cmd/check-l2/main.go

```diff
@@ -754,6 +754,34 @@ func checkL2StandardBridge(addr common.Address, client *ethclient.Client) error
 	}
 
 	log.Info("L2StandardBridge version", "version", version)
+
+	initialized, err := getInitialized("L2StandardBridge", addr, client)
+	if err != nil {
+		return err
+	}
+	log.Info("L2StandardBridge", "_initialized", initialized)
+	if initialized.Uint64() != 1 {
+		return fmt.Errorf("%w: %s", errInvalidInitialized, initialized)
+	}
+
+	abi, err := bindings.L2StandardBridgeMetaData.GetAbi()
+	if err != nil {
+		return err
+	}
+	calldata, err := abi.Pack("initialize", messenger, otherBridge)
+	if err != nil {
+		return err
+	}
+	if err := checkAlreadyInitialized(addr, calldata, client); err != nil {
```

Comment:
Can we use the implementation of `checkAlreadyInitialized` in each PR to remove the need for getting each individual initialized value and comparing it to 1?

Reply:
fixed